source/getting-started/hassbian-integrations.markdown

```diff
@@ -33,63 +23,8 @@ $ sudo raspi-config
 
 Select `Enable camera` choose `<Yes>` and hit `Enter`, then go to `Finish` and you'll be prompted to reboot.
 
-After reboot add your `homeassistant` account to the `video` group.
-
-```bash
-$ sudo adduser homeassistant video
-```
-
 After this follow the [Raspberry Pi Camera component](/components/camera.rpi_camera/) page.
 
-### {% linkable_title Raspberry Pi GPIO %}
-
-Each of the following devices are connected to the GPIO pins on the Raspberry Pi.
-For more details about the GPIO layout, visit the [documentation](https://www.raspberrypi.org/documentation/usage/gpio/) from the Raspberry Pi foundation.
-
```

Comment:
Instead of just deleting these, lets list them.

Keep the introduction lines above this comment and then:

Components and platforms that can use the Raspberry's GPIO pins include:
- [Raspberry Pi Cover component](/components/cover.rpi_gpio/)
- [DHT Sensor component](/components/sensor.dht/)
- [Raspberry PI GPIO Binary Sensor component](/components/binary_sensor.rpi_gpio/)
- etc...